### ISSUE TITLE
No POST request ?

### ISSUE THREAD

**Issue by Deksor** [jhhoward/MicroWeb#9]
I saw in the code that the browser is hardcoded to make "GET" requests. Will that remain forever or do you plan to let users make POST requests when they submit a form ?

**Comment by jhhoward** [comment]
Yes I am planning to add POST support, but as you mentioned it currently only supports GET

**Comment by LeJARB** [comment]
Why it doesn’t appear in release 2.1’s change log ?
Fortunately, i looked at updated issues 😁
